bild.py

```diff
@@ -42,6 +43,8 @@
 PYTHON3_TARGET	= "../antlr4-python3"
 CSHARP_TARGET	= "../antlr4-csharp"
 
+JAVA_VERSION = os.environ.get('JAVA_VERSION', None)
```

Comment:
ok, javac is piece of shit:

```
terence:master:~/github/bild $ javac -source 1.6 -target 1.6
javac: no source files
Usage: javac <options> <source files>
use -help for a list of possible options
terence:master:~/github/bild $ javac -source 1.7 -target 1.6
javac: source release 1.7 requires target release 1.7
```

why have -target if the damn thing won't gen earlier versions.  We have 1.7 code in some places, right?